src/script/scripting_client.cpp

```diff
@@ -51,7 +51,8 @@ ClientScripting::ClientScripting(Client *client):
 	InitializeModApi(L, top);
 	lua_pop(L, 1);
 
-	LuaMinimap::create(L, client->getMinimap());
+	if (client->getMinimap())
+		LuaMinimap::create(L, client->getMinimap());
```

Comment:
If the minimap is NULL at this time, client side scripting won't be able to access to `core.ui.minimap`, right?
Please add a note to `client_lua_api.md` that this table may be nil if the minimap is disabled.